src/commands/force/source/retrieve.ts

```diff
@@ -220,4 +229,56 @@ export class Retrieve extends SourceCommand {
     });
     return hasCustomField && !hasCustomObject;
   }
+
+  private async moveResultsForRetrieveTargetDir(): Promise<void> {
+    async function mv(src: string): Promise<string[]> {
+      let directories: string[] = [];
+      let files: string[] = [];
+      const srcStat = await fs.promises.stat(src);
+      if (srcStat.isDirectory()) {
+        const contents = await fs.promises.readdir(src);
+        directories = contents.filter((c) => fs.statSync(join(src, c)).isDirectory()).map((c) => join(src, c));
+        files = contents.filter((c) => !fs.statSync(join(src, c)).isDirectory());
+      } else {
+        files.push(src);
+      }
+      await promisesQueue(
+        files,
+        async (file: string): Promise<string> => {
+          const dest = join(src.replace(join('main', 'default'), ''), file);
+          const destDir = dirname(dest);
+          await fs.promises.mkdir(destDir, { recursive: true });
+          await fs.promises.rename(join(src, file), dest);
+          return dest;
+        },
+        50
+      );
+      return directories;
+    }
+
+    if (!this.flags.retrievetargetdir) {
+      return;
+    }
+    let overlapsProject = true;
+    const resolvedTargetDir = resolve(this.flags.retrievetargetdir as string);
```

Comment:
and what if I want to "peek" at the results, outside my normal project.  So I say, `--retrievetargetdir peek/main/default`

what's going to happen?  `.replace(join('main', 'default')` would eliminate the redundancy?

Reply:
I'll give that a try, but my suspect that it will only remove one `main/default`

Reply:
As predicted, the results land in `peek/main/default`